src/AudioSystem.cpp

```diff
@@ -74,9 +74,11 @@ bool AudioSystem::load_audio_assets() {
 }
 
 void AudioSystem::handle_keypress_event(const Event& e) {
-    int player_id = e.get_value<int>("player_id", -1);
-    int key = e.get_value<int>("key", -1);
-    int value = e.get_value<int>("value", 0);
+
+    int player_id = 0, key = 0, value = 0;
```

Comment:
looks better to have them all on separate lines, most coding standards ask to have one variable per line. I would also initialize these to -1 so that it's more obvious when they didn't get updated.